libs/nestjs-admin/src/adminUser.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, Column, PrimaryGeneratedColumn, BeforeInsert, BeforeUpdate } from 'typeorm'
+import { hashSync as bcryptHashSync } from 'bcryptjs'
+
+@Entity('adminUser')
+class AdminUser {
+  @PrimaryGeneratedColumn('uuid')
+  id: string
+
+  @Column({ length: 50 })
+  email: string
+
+  @Column({ length: 128 })
+  password: string
+
+  toString() {
+    if (this.email) {
+      return `${this.id} - ${this.email}`
+    }
+    return this.id
+  }
+
+  @BeforeInsert()
+  @BeforeUpdate()
+  hashPassword() {
+    this.password = bcryptHashSync(this.password)
```

Comment:
What if you save the entity multiple times? Does the encryption pile up?

Reply:
yes